controllers/machinehealthcheck_controller.go

```diff
@@ -256,12 +258,58 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	for _, t := range unhealthy {
 		condition := conditions.Get(t.Machine, clusterv1.MachineHealthCheckSuccededCondition)
 
-		if annotations.IsPaused(cluster, t.Machine) {
```

Comment:
Why is the IsPaused check being removed?

Reply:
TBH I had the same question as well. I looked at the [health checking proposal ](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20191030-machine-health-checking.md) but didn't immediately find any mention about pausing clusters or machines.